partials/examples/layouts.html

```diff
@@ -0,0 +1,27 @@
+<p>
+    This example shows default renderer for different layouts.
+</p>
+<p>
+    VerticalLayout:
+</p>
+<div class="example">
+    <listing-control schema="vm.schema" uischema="vm.uiSchemaVertical" data="vm.data"></listing-control>
+</div>
+<p>
+    HorizontalLayout:
+</p>
+<div class="example">
+    <listing-control schema="vm.schema" uischema="vm.uiSchemaHorizontal" data="vm.data"></listing-control>
+</div>
+<p>
+    Group:
+</p>
+<div class="example">
+    <listing-control schema="vm.schema" uischema="vm.uiSchemaGroup" data="vm.data"></listing-control>
+</div>
+<p>
+    Complex example:
```

Comment:
Explain the hierarchy of the layouts within the complex example.
Also, just an idea, but it would be cool, if we could color each layout in a different color for this single example in order to highlight the different containers.